test/FlexLabs.EntityFrameworkCore.Upsert.IntegrationTests/ContainerisedDatabaseInitializerFixture.cs

```diff
@@ -17,7 +17,7 @@ public abstract class ContainerisedDatabaseInitializerFixture<TBuilder, TContain
 
         protected static TBuilder ConfigureContainer(TBuilder builder)
             => builder
-                .WithName($"flexlabs_upsert_{DbDriverName.ToLowerInvariant()}")
+                .WithName($"flexlabs_upsert_{DbDriverName.ToLowerInvariant()}_{Environment.Version}")
```

Comment:
Looks good to me, but I'd add a small suggestion (to limit the number of containers created when updating minor versions of .NET and make the container name less confusing (i.e. it's the .NET version 9, not postgres 9)
```suggestion
                .WithName($"flexlabs_upsert_{DbDriverName.ToLowerInvariant()}_net{Environment.Version.Major}")
```